backend/app/services/adapters/task_kinds/queries.py

```
@@ -236,26 +242,96 @@ def get_user_group_tasks_lite(
+        # Filter tasks with is_group_chat=true in application layer
+        explicit_group_ids = set()
+        for row in candidate_tasks_result:
+            task_id, task_json = row
+            if task_id in member_task_ids:
+                continue  # Already counted
+            try:
+                if isinstance(task_json, dict):
+                    if task_json.get("spec", {}).get("is_group_chat") is True:
+                        explicit_group_ids.add(task_id)
+            except Exception:
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)